### PR TITLE
Add security page

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

File: pages/k8s/security.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  description: Using LDAP authentication via Keystone
+keywords: LDAP, Keystone, authorisation, AIM, SSH, Juju, users
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: security.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+
```
Comment (evilnick) [inline pages/k8s/security.md:16]: Needs an introduction

File: pages/k8s/security.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+The Kubernetes cluster deployed by **Charmed Kubernetes** can be secured using
+any of the methods and options described by the upstream
+[Kubernetes Security Documentation][].
+
+
+### Common tasks
+
+-   [Setting up vault as a CA][k8s-vault]
+
```
Comment (evilnick) [inline pages/k8s/security.md:68]: Are there any additional useful things we could link or add here?

**Comment by evilnick** [comment]
Preview here: https://deploy-preview-426--cdk-docs-next.netlify.app/kubernetes/docs/security

File: pages/k8s/security.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+-   [Configuring authorisation and authentication][k8s-auth]
+-   [Using AWS IAM for auth][k8s-aws-iam]
+-   [Authentication with LDAP and Keystone][k8s-ldap]
+-   [Overview of certificates and trust in Charmed Kubernetes][k8s-trust]
+
+## CIS Benchmark
+
+The **Center for Internet Security (CIS)** maintains a
+[Kubernetes benchmark][cis-benchmark] that is helpful to ensure clusters are
+deployed in accordance with security best practices. **Chamred Kubernetes**
```
Comment (tvansteenburgh) [inline pages/k8s/security.md:81]: Charmed

**Pull request merged**